dex/testing/loadbot/go.mod

```diff
@@ -69,12 +68,12 @@ require (
 	github.com/decred/go-socks v1.1.0 // indirect
 	github.com/decred/slog v1.2.0 // indirect
 	github.com/dgraph-io/badger v1.6.2 // indirect
-	github.com/dgraph-io/ristretto v0.0.2 // indirect
+	github.com/dgraph-io/ristretto v0.0.4-0.20210318174700-74754f61e018 // indirect
 	github.com/dustin/go-humanize v1.0.0 // indirect
-	github.com/edsrzf/mmap-go v1.0.0 // indirect
-	github.com/ethereum/go-ethereum v1.10.25 // indirect
+	github.com/edsrzf/mmap-go v1.1.0 // indirect
+	github.com/ethereum/go-ethereum v1.10.24-0.20220902154041-90711efb0ab6 // indirect
```

Comment:
v1.10.26 is actually latest, but it does not include the commit we need to use the webserver jwt auth. We need https://github.com/ethereum/go-ethereum/commit/90711efb0ab6360c37ab2135ab961daf05e83a74

I'm not sure why the go.mod wants v1.10.24, maybe it's the first common ancestor?

Reply:
>  Did they just now get to adding support to the client?

Yes. 

I gather the endpoint was created for connecting the beacon chain prism thing, but it connects over http, which is much simpler because you only need to send the auth token once in the initial header. With websocket you need to send it again on disconnects and there is no way to set that and use the provided client.

We could just use http for now and it would work with master.